packages/tools/src/tools/SculptorTool/CircleSculptCursor.ts

```diff
@@ -0,0 +1,174 @@
+import type { Types } from '@cornerstonejs/core';
+import { getEnabledElement } from '@cornerstonejs/core';
+import { ISculptToolShape } from './ISculptToolShape';
+import { distanceFromPoint, SculptData } from '../SculptorTool';
+import { drawCircle as drawCircleSvg } from '../../drawingSvg';
+import { point } from '../../utilities/math';
+import { SVGDrawingHelper, EventTypes, Annotation } from '../../types';
+
+export interface PushedHandles {
```

Comment:
This one should be a type - that is more generic in typescript, whereas interface is really only used when you need to declare extends ... or implements in a class.